eng/mgmt/mgmtmetadata/marketplace_resource-manager.txt

```diff
@@ -4,11 +4,11 @@ Commencing code generation
 Generating CSharp code
 Executing AutoRest command
 cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/marketplace/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\repo\azure-sdk-for-net\sdk
```

Comment:
You also need to update the csproj file and ```properties\AssemblyInfo.cs``` to reflect a new version (if the previous version is not a preview, this should be a major version bump). 

Otherwise, this looks good.

Reply:
we are still in public preview and no major bump up is needed.